src/arns.lua

```diff
@@ -21,8 +50,8 @@ NameRegistry = NameRegistry or {
 --- @param timestamp number The current timestamp
 --- @param processId string The process id
 --- @param msgId string The current message id
---- @param fundFrom string|nil The intended payment sources; one of "any", "balance", or "stakes". Default "balance"
---- @return table The updated record
+--- @param fundFrom string|nil The intended payment sources; one of "any", "balance", or "stake". Default "balance"
+--- @return BuyRecordResponse The response
```

Comment:
```suggestion
--- @return BuyRecordResponse buyRecordResponse The response
```

Reply:
addressed a good chunk in [cebb3cb](https://github.com/ar-io/ar-io-network-process/pull/175/commits/cebb3cba577aa5a936939eb7bc99e01da74ebb9f)